com.unity.formats.fbx/Editor/FbxExportSettings.cs

```diff
@@ -863,8 +868,8 @@ internal void LoadDefaults()
             prefabSavePaths = new List<string>(){ kDefaultSavePath };
             fbxSavePaths = new List<string> (){ kDefaultSavePath };
             integrationSavePath = DefaultIntegrationSavePath;
-            dccOptionPaths = null;
-            dccOptionNames = null;
+            dccOptionPaths = new List<string>();
+            dccOptionNames = new List<string>();
```

Comment:
Changed because tests were failing. `dccOptionPaths` and `dccOptionNames` are initialized to string lists, so resetting made sense to be string lists  unless they need to be null for another reason?